redbox/export/docx.py

```diff
@@ -18,11 +19,14 @@ def lookup_indentedness(raw: str, line_str_to_match: str):
             return len(line) - len(line.lstrip(" "))
 
 
-def summary_complete_to_docx(
-    summary_complete: SummaryComplete,
-    files: list[File],
+def summary_tasks_to_docx(
+    tasks: Iterable[SummaryTaskComplete],
+    reference_files: Iterable[File],
     title: Optional[str] = None,
-):
+    created: Optional[datetime] = datetime.now(),
```

Comment:
```suggestion
    created: Optional[datetime] = None,
```
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments